app/components/ui/button.tsx

```diff
@@ -58,13 +58,18 @@ const Button = React.forwardRef<HTMLButtonElement, ButtonProps>(
     const Comp = asChild ? Slot : 'button';
     return (
       <Comp
-        className={cn(buttonVariants({ variant, size, className }))}
+        className={cn('relative', buttonVariants({ variant, size, className }))}
```

Comment:
I don't think adding this style here should break something because I don't think we have absolutely positioned elements inside our app buttons atm but please correct me if I am wrong